README.md

```diff
@@ -0,0 +1,68 @@
+# Recipe App
+
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
+
+
+## Screenshot
+
+Home Page
+
+![](https://i.imgur.com/3kgpFmr.png)
+## Getting Started
+
+- Make sure you have [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Postgresql](https://www.postgresql.org/download/) installed on your system.
+
+- Install rails by running `gem install rails`
+
+- Clone this repository by running `git clone https://github.com/DammyShittu/Recipe-App.git` in your command line.
+
+- Navigate to the repository by running `cd Recipe-App`.
+
+- Run `yarn install` to install all dependencies
+
+- Run `rails db:create`, and `rails db:migrate`  to create and migrate your database.
+
+- Run `rails s` and go to http:localhost:3000 to view the app.
```

Comment:
[OPTIONAL]

- I would recommend adding a command to test the application. [ `rspec`]. As this is a small change, I'm expecting that you will add this in your README, hence I'm approving this pull request.